3f/pygmentize/2016-11-05.yaml

```diff
@@ -0,0 +1,9 @@
+title: Remote Code Execution
+link: https://github.com/dedalozzo/pygmentize/issues/1
+cve: ~
+branches:
+    master:
+        time: 2016-12-31 00:00:00
+        versions: ['<=1.1']
```

Comment:
if it is not fixed upstream yet, such constraint is wrong, as you are not sure which version will include the patch.